plugin-cgroupv2/src/k8s/plugin.rs

```diff
@@ -64,7 +71,7 @@ impl AlumetPlugin for K8sPlugin {
     fn start(&mut self, alumet: &mut alumet::plugin::AlumetPluginStart) -> anyhow::Result<()> {
         let v2_used: bool = super::utils::is_accessible_dir(&PathBuf::from("/sys/fs/cgroup/"));
         if !v2_used {
-            anyhow::bail!("Cgroups v2 are not being used!");
+            anyhow::bail!("Cgroup v2 are not being used!");
```

Comment:
En fait avec le `s` c'était pas si mal, ce sont _les_ control groups du noyau après tout.

Reply:
Okay, my bad, gonna put it back